src/message_bus.zig

```diff
@@ -709,7 +709,18 @@ fn MessageBusType(comptime process_type: vsr.ProcessType) type {
                     return null;
                 }
 
-                const header: *const Header = @alignCast(mem.bytesAsValue(Header, data[0..@sizeOf(Header)]));
+                // If the message bus receives more than one message at a time, subsequent messages
+                // might not be aligned to 16. These messages would be copied over to a fresh
+                // `Message` anyway, fixing the alignment issue, but care must be taken when
+                // working with the header before that.
+                var header_aligned: Header = undefined;
+                const header: *const Header = // if (connection.recv_parsed % @alignOf(Header) == 0)
+                    if (false)
```

Comment:
Will remove before the merge, but want to run our test-suite with the copying code to smoke test it